qa/L0_http/test.sh

```diff
@@ -367,6 +367,34 @@ if [ `grep -c "\[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0\]" ./curl.out` != "1" ]; then
     RET=1
 fi
 
+# send bad request with invalid json
```

Comment:
The json itself is not invalid. Fix:
Send bad request where the 'data' field misaligns with the 'shape' field of the input